.travis.yml

```diff
@@ -15,10 +15,11 @@ matrix:
   # This represents the minimum Rust version supported by Tokio. Updating this
   # should be done in a dedicated PR and cannot be greater than two 0.x
   # releases prior to the current stable.
-  - rust: 1.26.0
+  - rust: 1.28.0
```

Comment:
Is there a particular reason to bump the minimum stable? Does it matter that we *also* test on older stables? Should we consider also adding the intermediate stable (0.29) to CI?

Reply:
The reason to bump it was that I was in the file, and 1.26.0 no longer meets the minimum version support policy. Not sure if @carllerche cares about testing the intermediate versions, but it seems reasonable to just add, so I'll do that.

Reply:
I think testing the intermediate makes a lot of sense given the support policy. I don't know if we actively want to prune old versions "just because", though @carllerche may have contrary thoughts on that.

Reply:
I added the intermediate version in d677d60 so I'm gonna resolve this.

Reply:
I think you mean d7d6aa893cbe0f5b5529eb7cd1b1eb82196504c4 ?